Transformer/LuceneToBddTransformer.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace OpenOrchestra\Transformer;
+
+use OpenOrchestra\Transformer\LuceneToBddTransformerInterface;
+use OpenOrchestra\Exception\MissingFieldTransformerException;
+
+/**
+ * Class LuceneToBddTransformer
+ */
+class LuceneToBddTransformer implements LuceneToBddTransformerInterface
+{
+    protected $field = null;
+
+    /**
+     * @param string $field
+     */
+    public function setField($field)
+    {
+        $this->field = $field;
```

Comment:
I think there's a problem here if the service is called in two different contexts, each setting a field. This would lead to a collision. @nicolasThal: what do you think about this ? maybe use a prototype scope instead of a default container scope ?